core/controller.c

```diff
@@ -136,6 +138,48 @@ static void adapter_dump_state(int adapter)
   gstatus("\n");
 }
 
+static float float_swap(float x)
+{
+  union V {
+    float f;
+    uint32_t i;
+  };
+  union V val;
+  val.f = x;
+  val.i = htonl(val.i);
+  return val.f;
+}
+
+static void handlePacketConfig(const s_network_packet_config* buf)
+{
+  float s = float_swap(buf->sensibility);
+  if (s < FLT_MAX && s >= 0) {
+    printf("setting sensibility=%f\n", s);
+    cal_setSensibility(s);
+  }
+  int16_t dx = ntohs(buf->dead_zone_X);
+  if (dx < INT16_MAX) {
```

Comment:
Check look useless.

Reply:
The value INT16_MAX is used to signal for no modification.